web/models.py

```
@@ -2804,3 +2804,193 @@ class ScheduledPost(models.Model):
+    
+    # Optional social links
+    website = models.URLField(blank=True)
+    discord_link = models.URLField(blank=True)
+    facebook_link = models.URLField(blank=True)
+    twitter_link = models.URLField(blank=True)
+    
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    
+    class Meta:
+        ordering = ['-is_featured', 'name']
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)